wgpu-core/src/command/bundle.rs

```diff
@@ -732,18 +730,19 @@ pub type RenderBundleDescriptor<'a> = wgt::RenderBundleDescriptor<Label<'a>>;
 //Note: here, `RenderBundle` is just wrapping a raw stream of render commands.
 // The plan is to back it by an actual Vulkan secondary buffer, D3D12 Bundle,
 // or Metal indirect command buffer.
+#[derive(Debug)]
 pub struct RenderBundle<A: HalApi> {
     // Normalized command stream. It can be executed verbatim,
     // without re-binding anything on the pipeline change.
     base: BasePass<RenderCommand>,
     pub(super) is_depth_read_only: bool,
     pub(super) is_stencil_read_only: bool,
-    pub(crate) device_id: Stored<id::DeviceId>,
+    pub(crate) device_id: id::Valid<id::DeviceId>,
```

Comment:
I'm surprised that `RenderBundle` doesn't need a strong reference to its `Device` any more. I would expect `Global::render_bundle_drop` to crash if the `Device` gets dropped before the `RenderBundle`.